sdk/client_test.go

```diff
@@ -0,0 +1,67 @@
+package sdk
+
+import (
+	"net/http"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+const (
+	authHeader = "Authorization"
+	authPrefix = "Bearer "
+)
+
+func TestHeaders_Add(t *testing.T) {
+	t.Parallel()
+
+	Convey("given the sdk Headers struct contains a value for ETag", t, func() {
```

Comment:
Then Given, When, and Then clauses are usually initcapped. I thought that was a standard?